sdk/formrecognizer/azure-ai-formrecognizer/CHANGELOG.md

```diff
@@ -10,13 +10,16 @@
 - Removed `get_form_training_client` from `FormRecognizerClient`
 - Added `get_form_recognizer_client` to `FormTrainingClient`
 - A `HttpResponseError` is now raised if a model with `status=="invalid"` is returned from the `begin_train_model()` or `train_model()` methods
+- `PageRange` is renamed to `FormPageRange`
+- `FormField` does not have a page_number.
+- `begin_recognize_receipts` APIs now return `RecognizedReceipt` instead of `USReceipt`
```

Comment:
you also need to update the readme here: https://github.com/Azure/azure-sdk-for-python/blob/42684b2051480f546a4fd563411e9785a7d075f3/sdk/formrecognizer/azure-ai-formrecognizer/README.md#formrecognizerclient to say that it returns list of `RecognizedReceipt` instead of `USReceipt`

Reply:
nice catch - thanks